css/components/floating-ui.css

```
@@ -56,6 +56,178 @@
-.quote-tab:hover { 
-    background-color: #f90784; 
+.quote-tab:hover {
+    background-color: #f90784;
+}
+
+/* Help / Page Sidebar floating button — positioned above the quote tab */
+.help-tab {
+    position: fixed;
+    top: 28%;
+    right: 0;
+    transform: translateY(-50%) rotate(270deg);
```

Stylelint (17.2.0)

[error] Unexpected duplicate "transform" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)